packages/components/src/higher-order/use-filters/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useCallback, useState } from '@wordpress/element';
+import { addAction, applyFilters, removeAction } from '@wordpress/hooks';
+
+const useFilters = ( hookName, originalValue, ...args ) => {
+	const namespace = 'core/use-filter/' + hookName;
```

Comment:
The [Block Editor Coding Guidelines](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#strings) recommend using template strings instead of string concatenation whenever possible:
```suggestion
	const namespace = `core/use-filter/${ hookName }`;
```

Reply:
d92d8ee